benchmarks/src/move_only_function.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <benchmark/benchmark.h>
+#include <functional>
+
+using namespace std;
+
+void mof_none(benchmark::State& state) {
+    move_only_function<void()> mof;
+    for (auto _ : state) {
+        benchmark::DoNotOptimize(mof);
+    }
+}
+
+void mof_construct(benchmark::State& state) {
+    for (auto _ : state) {
+        move_only_function<void()> mof;
+        benchmark::DoNotOptimize(mof);
+    }
+}
+
+void mof_move(benchmark::State& state) {
+    move_only_function<void()> mof;
+    for (auto _ : state) {
+        auto mof_move = move(mof);
+        benchmark::DoNotOptimize(mof_move);
+    }
+}
+
+void mof_construct_and_move(benchmark::State& state) {
+    for (auto _ : state) {
+        move_only_function<void()> mof{};

```

Comment:
This line uses empty braces, unlike the previous constructions. As `move_only_function` has a constructor, the braces are unnecessary, and I think we can drop them here. I checked, and this makes no difference to the benchmark.